portfolio/src/main/webapp/script.js

```diff
@@ -25,31 +27,38 @@ $(document).ready(function () {
 });
 
 const animateScrollTop = (targetClass) => {
-  $("html").animate(
-    {
+  $("html").animate({
       scrollTop: $("." + targetClass).offset().top,
     },
     2000
   );
 };
 
+async function getComments() {
+  const response = await fetch("/data", {
+    method: "GET"
+  });
+  const jsonResponse = await response.text();
+  const jsonComments = JSON.parse(jsonResponse);
+  const comments = jsonComments["comments"];
+  for (let index = 0; index < comments.length; index++) {
+    const comment = comments[index];
+    $('#comments-container').append(`<p>${comment.comment}</p>`);
+    $('#comments-container').append(`<p>${comment.name}</p>`);
+  }
+}
+
 async function postComment(comment, name) {
-  const data = { name: name, comment: comment };
+  const data = {
+    name: name,
+    comment: comment
+  };
   const response = await fetch("/data", {
     method: "POST",
     body: JSON.stringify(data),
   });
-  const jsonComments = await response.text();
-  const comments = JSON.parse(jsonComments);
-  const commentNode = createNode("p", comments["comment"]);
-  document.getElementById("comments-container").appendChild(commentNode);
-  const nameNode = createNode("p", comments["name"]);
-  document.getElementById("comments-container").appendChild(nameNode);
-}
-
-function createNode(tag, text) {
-  const node = document.createElement(tag);
-  const textnode = document.createTextNode(text);
-  node.appendChild(textnode);
-  return node;
+  const jsonComment = await response.text();
+  const commentObj = JSON.parse(jsonComment);
+  $('#comments-container').append(`<p>${commentObj.comment}</p>`);
+  $('#comments-container').append(`<p>${commentObj.name}</p>`);
```

Comment:
FYI: we don't really need to parse the response because you're not really doing anything special with the response like handling errors, so you could just use the local data

```
$('#comments-container').append(`<p>${comment}</p>`);
$('#comments-container').append(`<p>${name}</p>`);
```

Reply:
Got brain dead there lol, done